installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc

```diff
@@ -1,13 +1,13 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="install-secured-cluster-config-options-ocp"]
-= Optional - Configuring Secured cluster configuration options for RHACS using the Operator
+= Configuring secured cluster configuration options for RHACS using the Operator
```

Comment:
Nit: capitalize `secured cluster` to make it consistent with `Central`

Reply:
From doing a quick search of the repo, we've been inconsistent about this. Central is always capitalized as a service, but Secured Cluster services aren't always capitalized (but I think they should be if we're referring to the services, and not capitalized if we're just referring to a cluster that has been secured). I'll fix it here and open a separate PR to get it sorted out throughout the repo after 4.4 goes out.